### PR TITLE
expose metrics

### PR DESCRIPTION
**Pull request by wheresrhys**
Don't want people requiring next-metrics directly as it will double up on the system performance stats @matthew-andrews

### CODE REVIEW

**Comment by wheresrhys** [comment]
maybe it should throw rather than warn?

File: main.js [hunk 1]
```diff
@@ -101,6 +101,12 @@ module.exports = function(options) {
 		});
 	}
 
+	if (require(path.join(process.cwd(), 'package.json')).dependencies['next-metrics']) {
```
Comment (matthew-andrews) [inline main.js]: hmm if we are going to do this we should do this everywhere……

Comment (matthew-andrews) [inline main.js]: oh actually if you're using `require` it uses `path.join` internally so it's not necessary

Comment (matthew-andrews) [inline main.js]: also i think `packageJson` is already in scope

File: main.js [hunk 2]
```diff
@@ -101,6 +100,13 @@ module.exports = function(options) {
 		});
 	}
 
+	if (packageJson.dependencies['next-metrics']) {
```
Comment (matthew-andrews) [inline main.js]: can you just put the var packageJson a little higher up this file so that this won't syntax error if there's no package.json file?

Comment (matthew-andrews) [inline main.js]: before the try { 
`var packageJson = {};`

**Pull request merged**